From: =?UTF-8?q?Carl=20F=C3=BCrstenberg?= <carl@excito.com>
Date: Mon, 21 Mar 2011 13:30:57 +0100
Subject: [PATCH] Remove max Audio::Scan version

The maximum version specifier for Audio::Scan has shown problematic in
some instances
---
 modules.conf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/modules.conf b/modules.conf
index 0a17d4b..8598906 100644
--- a/modules.conf
+++ b/modules.conf
@@ -5,7 +5,7 @@
 # <module> <min_version> [ <max_version> ]
 
 AnyEvent 5.202
-Audio::Scan 0.84 0.84
+Audio::Scan 0.84
 Cache::Cache 1.04
 CGI::Cookie 1.27
 Class::Data::Inheritable 0.04
-- 
